test/utils.js

```diff
@@ -268,3 +268,25 @@ export function expectNoErrorsFromResult(res) {
   res.errors && console.error(res.errors);
   expect(res.errors).to.not.exist;
 }
+
+/**
+ * This callback type is called `propertyCallback` and is called for every property of an object.
+ *
+ * @callback propertyCallback
+ * @param {string} key - Object key.
+ * @param {*} value - Object value at the given keys.
+ */
```

Comment:
I don't understand what this comment stands for

Reply:
This is how jsdoc recommends commenting callback functions https://jsdoc.app/tags-param.html#callback-functions. But since the comment intention was not clear I am inclined to remove it.

Reply:
@Betree I tried to improve the callback definition comment a little bit. Let me know what you think about it know 🙂